exercises/nucleotide-count/canonical-data.json

```diff
@@ -39,12 +71,38 @@
           }
         },
         {
-          "description": "strand with invalid nucleotides",
+          "description": "returns the count of nucleotides in a strand containing mixed-case nucleotides.",
           "property": "nucleotideCounts",
-          "strand": "AGXXACT",
+          "strand": "aGcTtTtCaTtCtGaCtGcAaCgGgCaAtAtGtCtCtGtGtGgAtTaAaAaAaGaGtGtCtGaTaGcAgC",
           "expected": {
-            "error": "Invalid nucleotide in strand"
+            "A": 20,
+            "C": 12,
+            "G": 17,
+            "T": 21
           }
+        },
+        {
+          "comments" : [
+            "This should call the property/function at least two times and check if it returns the same",
+            "result. This prevents errors where the call aggregates (sums up) the counts of each call."
+          ],
+          "description": "returns the same amounts for consecutive nucleotideCounts calls.",
+          "property": "nucleotideCounts",
+          "strand": "AGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTCTGATAGCAGC",
+          "expected" : true
+        },
+        {
+          "description": "handles incorrect inputs.",
+          "cases": [
+            {
+              "description": "throws error if DNA input does not include anything but ACGT-characters.",
```

Comment:
Some languages do not have exceptions, or would return an error type for invalid inputs.

Reply:
But this is not an issue with this canonical data.
"error" is explicitly described in the readme:
https://github.com/exercism/problem-specifications#test-data-format-canonical-datajson
and also in the schema.

Or do you just mind the wording of the description: "throws error"?
Not sure how to describe this instead.

Reply:
I simply object to the use of the word "throws", because it is not language general enough.  Instead, you might write "gives an error if.." or "xxx input is invalid".

Reply:
@stevejb71 thanks!
If you don't mind, there is also a discussion about this over here: #905